docs/how-to-guides/working-with-images.md

```diff
@@ -83,6 +83,26 @@ come to expect.
 ```console
 nerdctl build -t TAG .
 ```
+
+### Building Local Images
+
+To run your locally-built container images in Kubernetes manifests with `nerdctl`:
+
+```
+# nerdctl --namespace k8s.io build -t foo /some-dockerfile-directory
+# kubectl apply -f - <<EOF
```

Comment:
```suggestion
$ nerdctl --namespace k8s.io build -t foo /some-dockerfile-directory
$ kubectl apply -f - <<EOF
```

The source info used `#` prompts to signify that the commands must be run as `root` (inside the VM). Since the docs are about running them on the host, we should use a regular command prompt.